src/client/ui/EditorContainer.js

```diff
@@ -261,6 +260,11 @@ class EditorContainer extends Component {
   };
 
   serializeAndSaveScene = async sceneURI => {
+    this.showDialog(ProgressDialog, {
+      title: "Saving Scene",
+      message: "Saving scene..."
+    });
```

Comment:
This flashes the dialog briefly, since the save operation is usually quick after the first save.

Reply:
Yeah, what's the best way to handle this? Should we remove it entirely and just hide the dialog as soon as you confirm? Maybe we need to keep track of whether or not you are currently saving and cancel the current save action?

Reply:
I don't think it needs to be complicated. Usually you'd do something like:
```javascript
let saved = false;
setTimeout(() => {
  if (saved) return;
  this.showDialog(ProgressDialog, {
    title: "Saving Scene",
    message: "Saving scene..."
  });
}, 100);
```
and then set `saved = true` at the bottom of the `try`

I don't know if we need to do more to communicate that it _was_ successfully saved, any more than just the `*` disappearing from the title bar.